include/alpaka/kernel/KernelBundle.hpp

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2022 Benjamin Worpitz, Bert Wesarg, René Widera, Sergei Bastrakov, Bernhard Manfred Gruber
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include <alpaka/core/RemoveRestrict.hpp>
+
+#include <tuple>
+#include <type_traits>
+
+namespace alpaka
+{
+    //! \brief The class used to bind kernel function object and arguments together. Once an instance of this class is
+    //! created, arguments are not needed to be separately given to functions who need kernel function and arguments.
+    //! \tparam TAcc The accelerator type.
+    //! \tparam TKernelFn The kernel function object type.
+    //! \tparam TArgs Kernel function object invocation argument types as a parameter pack.
+    template<typename TAcc, typename TKernelFn, typename... TArgs>
```

Comment:
I know I wrote this object but after looking again on the code I think we should remove Acc from this object.